scripts/start.js

```diff
@@ -2,6 +2,22 @@
 process.env.BABEL_ENV = 'development';
 process.env.NODE_ENV = 'development';
 
+const BASE_URL = "http://localhost:3000";
+if (!process.env.REACT_APP_GQL_ENDPOINT)
+  process.env.REACT_APP_GQL_ENDPOINT = 'https://paas.beta.gccollab.ca/';
+if (!process.env.REACT_APP_OIDC_AUTHORITY)
+  process.env.REACT_APP_OIDC_AUTHORITY = 'https://dev.account.gccollab.ca/openid';
+if (!process.env.REACT_APP_OIDC_CLIENT_ID)
+  process.env.REACT_APP_OIDC_CLIENT_ID = 123;
```

Comment:
The login will also need the client_secret. So we will need to add a client_secret env string

Reply:
Since this is a front-end app, it's a different oidc flow that doesn't use a secret since it wouldn't exactly be secret.